app/javascript/packs/articleModerationTools.js

```diff
@@ -2,29 +2,31 @@
 import { isModerationPage } from '@utilities/moderation';
 
 getCsrfToken().then(() => {
-  const user = userData();
-  const { authorId: articleAuthorId, path } = document.getElementById(
-    'article-show-container',
-  ).dataset;
+  const articleContainer = document.getElementById('article-show-container');
 
-  const initializeModerationsTools = async () => {
-    const { initializeActionsPanel } = await import(
-      '../actionsPanel/initializeActionsPanelToggle'
-    );
-    const { initializeFlagUserModal } = await import('./flagUserModal');
+  if (articleContainer) {
+    const user = userData();
+    const { authorId: articleAuthorId, path } = articleContainer.dataset;
 
-    // article show page
-    if (user?.trusted) {
-      if (user?.id !== articleAuthorId && !isModerationPage()) {
-        initializeActionsPanel(user, path);
-        initializeFlagUserModal(articleAuthorId);
-        // dev.to/mod
-      } else if (isModerationPage()) {
-        initializeActionsPanel(user, path);
-        initializeFlagUserModal(articleAuthorId);
+    const initializeModerationsTools = async () => {
+      const { initializeActionsPanel } = await import(
+        '../actionsPanel/initializeActionsPanelToggle'
+      );
+      const { initializeFlagUserModal } = await import('./flagUserModal');
+
+      // article show page
+      if (user?.trusted) {
+        if (user?.id !== articleAuthorId && !isModerationPage()) {
+          initializeActionsPanel(user, path);
+          initializeFlagUserModal(articleAuthorId);
+          // dev.to/mod
```

Comment:
Quick, non-blocking suggestion: maybe we could change this to the following or something similar (`"/mod" route`) to generalize this comment a bit?:
```suggestion
          // "/mod" page
```

Reply:
💯